5.validate_model/validate_model.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# Convert notebook to python file and execute
+jupyter nbconvert --to python \
+        --FilesWriter.build_directory=scripts/nbconverted \
```

Comment:
Consider using the nbconvert `output-dir` alias for this line. While this performs the same action, it may increase readability.

```suggestion
        --output-dir=scripts/nbconverted \
```